.github/workflows/crossplane-integration-tests.yaml

```diff
@@ -0,0 +1,63 @@
+name: Crossplane Integration Tests
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - 'control-plane-crossplane/compositions/**/*.yaml'
+      - 'control-plane-crossplane/compositions/**/*.xrd.yaml'
+      - 'control-plane-crossplane/compositions/**/*.workload-cluster-composition.yaml'
+      - 'control-plane-crossplane/compositions/**/*.workload-cluster-xrd.yaml'
```

Comment:
We can consolidate all these paths and assume that any change in "compositions" folder requires this CI workflow.